handbook/product/README.md

```diff
@@ -307,22 +307,23 @@ We track competitors' capabilities and adjacent (or commonly integrated) product
 ## Intake
 
 - [Making a request](#making-a-request)
-- [How features are evaluated](#how-features-are-evaluated)
+- [How feature requests are evaluated](#how-feature-requests-are-evaluated)
 - [After the feature is accepted](#after-the-feature-is-accepted)
 - [Why this way?](#why-this-way)
 
 To stay in-sync with our customers' needs, Fleet accepts feature requests from customers and community members on a sprint-by-sprint basis in the [regular 🎁🗣 Feature Fest meeting](#rituals). Anyone in the company is invited to submit requests or simply listen in on the 🎁🗣 Feature Fest meeting. Folks from the wider community can also [request an invite](https://fleetdm.com/contact). 
 
 ### Making a request
 
-To make a feature request or advocate for a feature request from a customer or community member, add your request to the list in the [🎁🗣 Feature Fest agenda](https://docs.google.com/document/d/1mwu5WfdWBWwJ2C3zFDOMSUC9QCyYuKP4LssO_sIHDd0/edit#heading=h.zahrflvvks7q) and attend the next scheduled 🎁🗣 Feature Fest meeting. 
+To make a feature request or advocate for a feature request from a customer or community member, [create an issue](https://github.com/fleetdm/fleet/issues/new?assignees=&labels=~customer+request&projects=&template=feature-request.md&title=) and attend the next scheduled 🎁🗣 Feature Fest meeting. 
 
 Requests are weighed from top to bottom while prioritizing attendee requests. This means that if the individual that added a feature request is not in attendance, the feature request will be discussed towards the end of the call if there's time.
 
 To be acceptable for consideration, a request must:
 - Have a clear proposed change
 - Have a well-articulated underlying user need
 - Specify the requestor (either internal stakeholder or customer or community user)
+- Include the `feature fest` and customer labels (if applicable) 
```

Comment:
I'd put this thing about the feature fest label some where else. Like this section is about how to write a good request. then a separate section should be about how do you get it onto the feature fest board.

1. add the label
2. here's a link to the board

Reply:
Good idea! Have broken this down into a new section and will do a new PR to add the link the board once Luke and I create it on Monday.